Tutorials/how-to.md

```diff
@@ -9,71 +9,70 @@ This guide provides links to tutorial solutions as quick references for seL4 cal
 
 [//]: # (<html><a href="no-javascript.html" title="Get some foo!">Show me some foo</a></html>)
 
-
 ## The seL4 kernel
 
-### [Capabilities](capabilities.md)
+### [Capabilities](capabilities?tut_expand)
 
  - [Calculate the size of a CSpace](capabilities.md#how-big-is-your-cspace)
  - [Copy a capability between CSlots](capabilities.md#copy-a-capability-between-cslots)
  - [Delete a capability](capabilities.md#how-do-you-delete-capabilities)
  - [Suspend a thread](capabilities.md#suspend-a-thread)
```

Comment:
I'm afraid you need to add `?tut_expand` to all the links for it to work, now only the top level links expand the hints.

Maybe using a variable is nicer than all the repetitions of `capabilities?tut_expand`.

Reply:
It actually does work with just the top level links. The `js` code looks for everything that's a `details` markdown.

Reply:
Yes, everything gets expanded if you follow the top link. But if you follow one of the sublinks, `tut_expand` won't be part of the URL and that code does nothing.

Reply:
Blast, ok. I'll fix it on Wed.